src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser.g4

```diff
@@ -893,7 +893,9 @@ javadocTag: AUTHOR_LITERAL (WS | NEWLINE)* description?
 
       | VERSION_LITERAL (WS | NEWLINE)* description?
 
-      | CUSTOM_NAME (WS | NEWLINE)* description?
+      | CUSTOM_NAME (WS | NEWLINE)*
+        ({!isNextJavadocTag()}? LEADING_ASTERISK | htmlComment
+        | CDATA | NEWLINE | text | javadocInlineTag | htmlElement)*
```

Comment:
@romani 
you said that they should be affected https://github.com/checkstyle/checkstyle/pull/4423#discussion_r120632674
@ps-sp I don't think a new issue is needed. Please split changes for inline tags and block tags in separate PRs to speed up acceptance.